src/AdoNet/Orleans.Clustering.AdoNet/Microsoft.Orleans.Clustering.AdoNet.targets

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="MicrosoftOrleansClusteringAdoNetCopySqlFiles" BeforeTargets="PrepareForBuild">
```

Comment:
I don't know if there is a `AfterTargets` for Nuget restore, which might be better than this. On the other than, this restores files automatically when potentially needed.